data/Language/Rust.yaml

```diff
@@ -0,0 +1,79 @@
+identifier: Rust
+full_name: Rust
+wikidata: Q575650
+aliases:
+  - Rust
+  - rs
+extensions:
+  - rs
+  - rlib
+line_continuation: '\'
+delimiters:
+  # comment delimiters
+  - double_slash
+  - triple_slash
+  - doc_comment
+  - multiline_slash_star
+  # string delimiters:
+  - c_char
+  - double_quote_slash_escape
+  - curly_braces
+keywords:
+ - as
+ - use
+ - extern crate
+ - break
+ - const
+ - continue
+ - crate
+ - else
+ - if
+ - if let
+ - enum
+ - extern
+ - false
+ - fn
+ - for
+ - if
+ - impl
+ - in
+ - for
+ - let
+ - loop
+ - match
+ - mod
+ - move
+ - mut
+ - pub
+ - impl
+ - ref
+ - return
+ - Self
+ - self
+ - static
+ - struct
+ - super
+ - trait
+ - true
```

Comment:
true & false are a boolean datatype .  Please see other recently added languages for how to do this

Reply:
fixed